dvc/utils/hash/__init__.py

```diff
@@ -0,0 +1,114 @@
+"""Helpers for other modules."""
```

Comment:
From @shcheklein 's reply 4 days ago:
> should we move hashing related stuff to a separate module? so, that it looks like hash.CHECKSUM_MD5. utils should contain really generic stuff, everything else should be moving into packages with some meaningful names

now go back to utils again?

Reply:
But your `dvc/utils/hash/__init__.py` is still a part of utils, isn't it? :slightly_smiling_face: Don't quite remember the specifics, but @shcheklein probably meant that you shouldn't put all of that stuff in `dvc/utils/__init__.py` itself. We will probably move a bunch of those things

Reply:
Mostly, that stuff was sitting there before...  
I just collect same value multiple constants out there from other modules to reuse...
@shcheklein so do you like `utils.CHECKSUM_MD5`?

Reply:
@vyloy But what I ask for is not `utils.CHECKSUM_MD5`, but `dvc.utils.checksum.CHECKSUM_MD5`. Though, please see another comment about keeping all hash/checksum logic contained in RemoteLOCAL instead of keeping it somewhere in the utils.

Reply:
@efiop yep, I'm totally fine with `dvc.utils.checksum.CHECKSUM_MD5`. I'm not sure about `RemoteLOCAL` - will check the comment.